metrics/firebase/functions/test/main_test.dart

```diff
@@ -87,16 +96,9 @@ void main() {
 
         await onBuildAddedHandler(documentSnapshotMock, null);
 
-        final documentDataMatcher = predicate<DocumentData>((data) {
-          final operand =
-              data.getNestedData('successfulBuildsDuration').getInt('operand');
+        final matcher = _operandCountMatcher('successfulBuildsDuration', 0);
```

Comment:
```suggestion
        final successfulDurationIncrementMatcher = _operandCountMatcher('successfulBuildsDuration', 0);
```
Would it make tests a bit more readable? If so, please, apply this suggestion to all similar situations.